### PR TITLE
fix(popup): group finished campaigns under completed

### LINKED ISSUE #569: fix(popup): consolidate finished campaigns in a Completed section
## What happened?

Campaign rows can show several contradictory signals after every reward has already been completed:

- **100%** progress in the row headline
- a **Later** pill whose tooltip says the campaign will be farmed later
- a **Finished** lifecycle pill
- an amber warning whose tooltip explains why the campaign is not being watched

This makes a completed campaign look queued, finished, and blocked at the same time. The same noise repeats for every completed campaign and takes space away from campaigns that still require attention.

The underlying cause is that these signals are derived independently. **Later** is based on the campaign's position relative to the currently farmed campaign, while **Finished** and the rejection warning come from lifecycle and farmability evaluation.

## Steps to reproduce

1. Open the Lurkloot browser-extension popup.
2. Enable Twitch or Kick and populate the Drops list with multiple campaigns.
3. Complete all rewards in one or more campaigns while another campaign remains active and is being farmed.
4. Keep finished campaigns visible in the Drops list.
5. Observe a finished campaign above the currently farmed campaign.

## Current result

The finished row can simultaneously render `100%`, `Later`, `Finished`, and a farming-rejection warning.

## Expected result

Finished campaigns should have one clear terminal presentation and should not compete visually with campaigns that can still be farmed.

## Agreed design

- Split campaigns into an active priority list and a **Completed** subsection.
- Place the Completed subsection below active campaigns.
- Collapse Completed by default and show its campaign count in the disclosure header.
- When expanded, show **Finished** as the row's single headline state with a check icon.
- Do not show the following on a finished row:
  - `100%` headline progress
  - `Later`
  - lifecycle status pill duplicating `Finished`
  - farming-rejection warning or expanded rejection explanation
  - priority rank or drag handle
  - edge progress bar
- Keep reward details available by expanding the finished campaign row.
- Preserve the stored positions of finished campaigns when active campaigns are reordered, so their relative priority is retained if campaign state changes.
- Keep finished campaigns discoverable through Drops search.
- If another popup action focuses a finished campaign, automatically open the Drops section, open Completed, expand the campaign, and scroll it into view.

## Important edge case

A watch reward at **100%** that is still claimable or otherwise unclaimed is not a finished campaign. It must remain in the active list and retain its progress presentation.

## Acceptance criteria

- [ ] Completed is collapsed by default and displays the correct count.
- [ ] Active campaigns remain visible and keep contiguous visible ranks.
- [ ] Expanding Completed reveals all finished campaigns.
- [ ] Each collapsed finished row shows exactly one terminal status: **Finished**.
- [ ] Finished rows do not show `100%`, `Later`, rejection warnings, ranks, drag handles, or progress bars.
- [ ] A finished campaign's reward details can still be expanded.
- [ ] Reordering active campaigns preserves finished campaigns in the stored order passed to settings persistence.
- [ ] Search finds finished campaigns even while Completed is collapsed.
- [ ] Focus navigation reveals, expands, and scrolls to a finished campaign.
- [ ] A 100%-watched but unclaimed campaign stays in the active list.
- [ ] The Completed label is localized in every supported popup locale.
- [ ] The behavior is platform-independent for Twitch and Kick campaign views.

## Test coverage

Add focused popup tests for:

- completed disclosure default state and terminal row presentation
- the 100%-watched/unclaimed edge case
- active-rank persistence with a finished campaign between active campaigns
- search behavior while Completed is collapsed
- focus navigation into Completed
- existing completed subscription reward rendering under the new grouping

Run `pnpm check` and a production Chromium extension build.

## Environment

- Surface: browser-extension popup
- Platforms: Twitch and Kick
- Version observed: 1.13.0 development line
- Watch mode: not specific to a watch mode

Related: #565 covers localization of the `Later` label; this issue covers whether that state is semantically valid for finished campaigns.

### PR DESCRIPTION
**Pull request by jamezrin**
## Summary

- move finished campaigns out of the active priority queue and into a collapsed **Completed** subsection
- present each completed row with one **Finished** headline state while removing `100%`, `Later`, rejection warnings, priority controls, and the edge progress bar
- preserve stored campaign order when active campaigns are reordered, and keep completed campaigns reachable through search and focus navigation
- localize the new Completed section label across every supported popup locale

## Behavior details

- A reward at 100% that remains claimable or unclaimed stays in the active list.
- Opening a focused finished campaign automatically opens the Completed subsection, expands the campaign, and scrolls it into view.
- Finished campaign reward details remain available from the row disclosure.

## Testing

- `pnpm check`
  - script tests: 95 passed
  - CLI tests: 199 passed
  - extension tests: 2,278 passed
  - workspace typechecks passed
  - Astro site tests and builds passed
- `pnpm build`
  - Chromium MV3 production build passed

## Visual QA

Rendered the real popup UI in headless Chromium at the extension's popup dimensions and verified both disclosure states:

| State | Result |
| --- | --- |
| Default | Completed is collapsed, displays its count, and finished rows do not occupy the active queue. |
| Expanded | The row displays a check icon and **Finished** as its only terminal signal, with no warning indicator. |

No relevant page or application console errors were observed during the interaction.

Closes #569

## Summary by CodeRabbit

* **New Features**
  * Added a collapsible “Completed campaigns” section for finished campaigns.
  * Finished campaigns now display a simplified completed status without progress, ranking, or action controls.
  * Searching for or focusing a completed campaign automatically reveals it.
  * Active campaign reordering preserves the order of completed campaigns.

* **Localization**
  * Added “Completed campaigns” translations across supported languages.

* **Bug Fixes**
  * Improved campaign waiting-state handling when rewards are unavailable or farming is rejected.

### CODE REVIEW

File: packages/popup-ui/src/drops.tsx [hunk 1]
```diff
@@ -90,6 +98,7 @@ export function DropsPanel({ campaigns, gameMap, focus, refreshing, startCollaps
     setQuery("");
     setSearchOpen(false);
     setSectionExpanded(true);
+    if (campaigns.some((campaign) => campaign.id === focus.id && campaign.lifecycle === "finished")) setCompletedExpanded(true);
     setExpandedIds((current) => ({ ...current, [focus.id]: true }));
   }, [focus?.seq]);
```
[the remark this answers is not in the document]
Comment (jamezrin) [reply packages/popup-ui/src/drops.tsx]: Fixed in 5ba48ad9. The effect now depends on a primitive `focusedCampaignFinished` value, so it reruns when the focused campaign enters the finished lifecycle without subscribing to the full campaigns array. Added a regression test for the unchanged-focus-sequence transition.
